overlord/hookstate/hooks.go

```diff
@@ -333,20 +333,11 @@ func SetupGateAutoRefreshHook(st *state.State, snapName string) *state.Task {
 	return task
 }
 
-type snapHookHandler struct {
-}
-
-func (h *snapHookHandler) Before() error {
-	return nil
-}
+type SnapHookHandler struct{}
```

Comment:
do we need to export this now?

Reply:
The <plug>-view-changed hooks can use it. I could duplicate it but it this seemed reasonable